com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/FileSystemConfiguration.java

```diff
@@ -93,14 +95,14 @@
    */
   @Before
   public void buildTempFilesystem() {
-    intializeSettingsWithSomeContent();
+    intializeSettings();
 
     createWorkspaceFolderStructure();
     createCopybookFolders();
     createDependencyFolder();
 
+    // populate copybook folder and dependency file with some content
     createCopybookFiles();
-    createNestedCopybooks();
```

Comment:
Why this init for nested copybooks was removed?

Reply:
Ok I got it - cause when I tested the nested copybooks I simulate that the cobol file have a reference to another copybook without psychically stor e in the file. So, is fair that you removed it from the initial activities but please remove also the method definition at line 179